presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -189,7 +190,8 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         }
 
         columns.add(hiddenColumn(SHARD_UUID_COLUMN_NAME, VARCHAR));
-        return new ConnectorTableMetadata(tableName, columns);
+
+        return new ConnectorTableMetadata(tableName, columns, Collections.EMPTY_MAP, null, false, Optional.ofNullable(dao.getMaterializedQuery(handle.getTableId())));
```

Comment:
Always the functions like `emptyMap()` rather than the constants. And static import